Students/JuniorStudent.js

```diff
@@ -0,0 +1,138 @@
+const db = require("../db");
+const Borrow = require("../Borrow/Borrow.js");
+// Define a global variable to help with auto-incrementation of student ID
+global.counter = 1;
+
+//constructor function for creation of a junior student
```

Comment:
Give a space between comment symbol and comment.

Reply:
Also done